sql/mozfun/norm/truncate_version/udf.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE FUNCTION norm.truncate_version(os_version STRING, truncation_level STRING)
+RETURNS STRING AS (
+  CASE
+  WHEN
+    truncation_level = "minor"
+  THEN
+    REGEXP_REPLACE(os_version, r"^([0-9]+[.]?[0-9]+).*", "\\1")
```

Comment:
Behavior might be more inline with extract than replace:

```suggestion
    REGEXP_EXTRACT(os_version, r"^([0-9]+[.]?[0-9]+).*")
```